tests/test_wps_generate_climos.py

```diff
@@ -39,26 +62,46 @@
     ],
 )
 def test_wps_gen_climos_opendap(netcdf, kwargs):
-    client = client_for(Service(processes=[GenerateClimos()]))
-    datainputs = (
-        "netcdf=@xlink:href={0};"
-        "operation={operation};"
-        "climo={climo};"
-        "resolutions={resolutions};"
-        "convert_longitudes={convert_longitudes};"
-        "split_vars={split_vars};"
-        "split_intervals={split_intervals};"
-        "dry_run={dry_run};"
-    ).format(netcdf, **kwargs)
+    run_wps_generate_climos(netcdf, kwargs)
 
-    resp = client.get(
-        service="wps",
-        request="Execute",
-        version="1.0.0",
-        identifier="generate_climos",
-        datainputs=datainputs,
-    )
-    assert_response_success(resp)
+
+local_test_data = [
+    nc for nc in TESTDATA["test_local_nc"] if not nc.endswith("_climos.nc")
+]
```

Comment:
It would be helpful to place a comment explaining why we are filtering out certain items from the list.

Reply:
@nikola-rados I am thinking of writing an issue on refactoring `common.py` and work on it after my 2 PRs are merged. That way I think I could clean up the script much cleaner.

Reply:
Sorry I missed this comment! That sounds like a good idea.